### PR TITLE
🌍 add translations for `desktop-client/components/accounts`

### PR DESCRIPTION
**Pull request by matt-fidd**
Translation support for `packages/desktop-client/src/components/accounts`

### CODE REVIEW

File: packages/desktop-client/src/components/accounts/Reconcile.jsx [hunk 1]
```diff
@@ -59,33 +60,42 @@ export function ReconcilingMessage({
...
-              <strong>{format(cleared, 'financial')}</strong> needs{' '}
-              <strong>
-                {(targetDiff > 0 ? '+' : '') + format(targetDiff, 'financial')}
-              </strong>{' '}
-              to match
-              <br /> your bank’s balance of{' '}
-              <Text style={{ fontWeight: 700 }}>
-                {format(targetBalance, 'financial')}
-              </Text>
+              <Trans>
```
Comment (matt-fidd) [inline packages/desktop-client/src/components/accounts/Reconcile.jsx:68]: This is causing some sort of problem, but there's no error. For me, the reconcile window immediately closes.

Could anyone else confirm please?

File: packages/desktop-client/src/components/accounts/Header.jsx [hunk 2]
```diff
@@ -501,36 +506,39 @@ function AccountMenu({
       items={[
         isSorted && {
           name: 'remove-sorting',
-          text: 'Remove all sorting',
+          text: t('Remove all sorting'),
         },
         canShowBalances && {
           name: 'toggle-balance',
-          text: (showBalances ? 'Hide' : 'Show') + ' running balance',
+          text: (showBalances ? t('Hide') : t('Show')) + t(' running balance'),
```
Comment (julianwachholz) [inline packages/desktop-client/src/components/accounts/Header.jsx:513]: This is a good example for strings that should be translated as a whole. 

A translator who will only see `" running balance"` won't be able to make an informed decision about the translation without the "Show" or "Hide" context.

**Review by julianwachholz — CHANGES_REQUESTED** [review]

File: packages/desktop-client/src/components/accounts/Reconcile.jsx (hunk 1 above)
Comment (julianwachholz) [reply packages/desktop-client/src/components/accounts/Reconcile.jsx:68]: You can enable debug output in the console by setting `debug: true` in the options in `i18n.ts`.

File: packages/desktop-client/src/components/accounts/Header.jsx [hunk 3]
```diff
@@ -231,7 +234,7 @@ export function AccountHeader({
                   data-testid="account-name"
                 >
                   {account && account.closed
-                    ? 'Closed: ' + accountName
+                    ? t('Closed: ') + accountName
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/accounts/Header.jsx:237]: **💬 suggestion:** ‏should include the `accountName` as a variable here.

I'm not 100% sure about the specific syntax, but the idea is this:

```suggestion
                    ? t('Closed: {accountName}', { accountName })
```

File: packages/desktop-client/src/components/accounts/Header.jsx [hunk 4]
```diff
@@ -262,7 +265,7 @@ export function AccountHeader({
                 data-testid="account-name"
               >
                 {account && account.closed
-                  ? 'Closed: ' + accountName
+                  ? t('Closed: ') + accountName
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/accounts/Header.jsx:268]: same here

File: packages/desktop-client/src/components/accounts/Header.jsx (hunk 2 above)
Comment (MatissJanis) [reply packages/desktop-client/src/components/accounts/Header.jsx:513]: Agreed (for this and the other occurrences below as well)

File: packages/desktop-client/src/components/accounts/Reconcile.jsx [hunk 5]
```diff
@@ -59,33 +60,42 @@ export function ReconcilingMessage({
...
+                needs{' '}
+                <strong>
+                  {{
+                    difference:
+                      (targetDiff > 0 ? '+' : '') +
+                      format(targetDiff, 'financial'),
+                  }}
+                </strong>{' '}
+                to match
+                <br /> your bank&aposâs balance of{' '}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/accounts/Reconcile.jsx:82]: **❓ question:** ‏what's this?

```suggestion
                <br /> your bank's balance of{' '}
```

**Review by MatissJanis — CHANGES_REQUESTED** [review]

File: packages/desktop-client/src/components/accounts/Header.jsx (hunk 3 above)
Comment (julianwachholz) [reply packages/desktop-client/src/components/accounts/Header.jsx:237]: This example would require double curly braces (`'Closed: {{ accountName }}'`)

File: packages/desktop-client/src/components/accounts/AccountSyncCheck.jsx [hunk 6]
```diff
@@ -17,31 +20,37 @@ function getErrorMessage(type, code) {
...
-      return 'Your SimpleFIN Access Token is no longer valid. Please reset and generate a new token.';
+      return t(
+        'Your SimpleFIN Access Token is no longer valid. Please reset and generate a new token.',
+      );
 
     case 'ACCOUNT_NEEDS_ATTENTION':
       return (
         <>
-          The account needs your attention at{' '}
+          <Trans>The account needs your attention at</Trans>{' '}
```
Comment (julianwachholz) [inline packages/desktop-client/src/components/accounts/AccountSyncCheck.jsx:53]: This translation unit should also include the `<Link>` below to form a complete sentence.

File: packages/desktop-client/src/components/accounts/Reconcile.jsx (hunk 5 above)
Comment (psybers) [reply packages/desktop-client/src/components/accounts/Reconcile.jsx:82]: @MatissJanis I think it was a weird typo that already existed.  It is supposed to be "bank's" but somehow got converted into an HTML entity `&apos;` and then the semi was converted into an `â` somehow.

File: packages/desktop-client/src/components/accounts/Reconcile.jsx (hunk 1 above)
Comment (matt-fidd) [reply packages/desktop-client/src/components/accounts/Reconcile.jsx:68]: This persists in edge, not a symptom of this PR.

**Pull request merged**